### PR TITLE
[BPK-4379] Migrate all 'B' components to new story format

### PR DESCRIPTION
**Pull request by shaundon**

### CODE REVIEW

File: packages/bpk-component-badge/examples.js [hunk 1]
```diff
@@ -0,0 +1,187 @@
...
+  CenteredExample,
+  DockedLeadingExample,
+  DockedTrailingExample,
+  WarningExample,
+  SuccessExample,
+  DestructiveExample,
+  LightExample,
+  InverseExample,
+  OutlineExample,
+};
```
Comment (georgegillams) [inline packages/bpk-component-badge/examples.js:187]: Why don't we put the `export` keyword inline?
Like `export const OulineExample...`

File: packages/bpk-component-blockquote/examples.js [hunk 2]
```diff
@@ -0,0 +1,33 @@
...
+ * limitations under the License.
+ */
+
+import React from 'react';
+
+import BpkBlockquote from './index';
+
+const textContent = `
+It's your world and we'll help you explore it.
+Find the best prices across millions of flights, hotels and car hire options to create your perfect trip.`;
```
Comment (georgegillams) [inline packages/bpk-component-blockquote/examples.js:25]: Surely this shouldn't begin with a line-break?

File: packages/bpk-component-button/stories.js [hunk 3]
```diff
@@ -3,15 +3,15 @@
...
  * You may obtain a copy of the License at
  *
  *   http://www.apache.org/licenses/LICENSE-2.0
  *
- * Unless required by applicable law or agreed to in writing, software
+ * Unless required by applicable law or agreed to in writing, () => <, software />
  * distributed under the License is distributed on an "AS IS" BASIS,
- * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, () => <, either />  express or implied.
  * See the License for the specific language governing permissions and
```
Comment (georgegillams) [inline packages/bpk-component-button/stories.js:15]: what?

File: packages/bpk-component-blockquote/examples.js (hunk 2 above)
Comment (shaundon) [reply packages/bpk-component-blockquote/examples.js:25]: It doesn't make any difference in a template string

File: packages/bpk-component-button/stories.js (hunk 3 above)
Comment (shaundon) [reply packages/bpk-component-button/stories.js:15]: That's my find and replace regex gone awry 😂

File: packages/bpk-component-badge/examples.js (hunk 1 above)
Comment (shaundon) [reply packages/bpk-component-badge/examples.js:187]: I've just always preferred the neatness of doing it this way, I prefer it

File: packages/bpk-component-button/stories.js (hunk 3 above)
Comment (shaundon) [reply packages/bpk-component-button/stories.js:15]: Fixed

**Review by georgegillams — APPROVED** [review]

**Pull request merged**